website/src/components/astGrep/state.ts

```diff
@@ -36,40 +39,10 @@ export function deserialize(str: string): State {
   return JSON.parse(atou(str))
 }
 
-const source =
```

Comment:
I don't think moving it to standalone files make too much difference. I am okay with hardcoding these default templates.

Reply:
Maybe will use it somewhere else in the future, so I used a separate file.